### PR TITLE
chore: 🍰 New Digital Ocean Kubernetes Cluster

### LINKED ISSUE #4043: 💥 [DevOps] Setup A New Digital Ocean Cluster
## :fire: DevOps ticket

Setup a new digital ocean cluster.

### Motive

Have a new running network with real users.

### Related issues

Related issue #4010 .

### Implementation

I will find out.

### Validation

None

### Additional context

None

### PR DESCRIPTION
**Pull request by Tirokk**
### Issues

- relates #4043

### Todo

- [ ] a lot

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -22,10 +22,10 @@
   * [Digital Ocean](deployment/digital-ocean/README.md)
     * [Kubernetes Dashboard](deployment/digital-ocean/dashboard/README.md)
     * [HTTPS](deployment/digital-ocean/https/README.md)
-  * [Human Connection](deployment/human-connection/README.md)
-    * [Error Reporting](deployment/human-connection/error-reporting/README.md)
-    * [Mailserver](deployment/human-connection/mailserver/README.md)
-    * [Maintenance](deployment/human-connection/maintenance/README.md)
+  * [Human Connection](deployment/ocelot-social/README.md)
```
Comment (ulfgebhardt) [inline SUMMARY.md:25]:
```suggestion
  * [Ocelot Social](deployment/ocelot-social/README.md)
```

**Review by ulfgebhardt — APPROVED** [review]

File: SUMMARY.md (hunk 1 above)
Comment (ulfgebhardt) [reply SUMMARY.md:25]: i suggest to merge this, but not close #4043 yet

Comment (Tirokk) [reply SUMMARY.md:25]: This is an issue @Mogge works on separately …
Therefore I thought not to cross his lines … @ulfgebhardt

Comment (Tirokk) [reply SUMMARY.md:25]: Oh, this is the SUMMARY.md. Thought it would be the main `README.md`.
I have fixed it locally. @ulfgebhardt

**Review by Mogge — APPROVED** [review]
Wow! That is impressive. Thank you so much.

**Pull request merged**